File: app/src/main/java/de/thinksonic/basictemplate/Interface/VolleyHandler.java
package de.thinksonic.basictemplate.Interface;

import com.android.volley.VolleyError;
import org.json.JSONObject;


/*
    Created By : Shesha Vasukhi Prasad
    Date : 15-Jul-2019
    Time : 22:11
    For Project : BasicTemplate
*/

import de.thinksonic.basictemplate.Tools.RequestTypes;

public interface VolleyHandler {

    void onSuccess(RequestTypes requestTypes, JSONObject jsonObject);

    void onError(RequestTypes requestTypes, VolleyError volleyError);

}

File: app/src/main/java/de/thinksonic/basictemplate/interfaces/VolleyHandler.kt
package de.thinksonic.basictemplate.interfaces

import com.android.volley.VolleyError
import org.json.JSONObject


/*
    Created By : Shesha Vasukhi Prasad
    Date : 15-Jul-2019
    Time : 22:11
    For Project : BasicTemplate
*/

import de.thinksonic.basictemplate.tools.RequestTypes

/**
Created By : Shesha Vasukhi Prasad
Date : 16-Jul-2019
Time : 01:52
For Project : BasicTemplate
 */

interface VolleyHandler {

    fun onSuccess(requestTypes: RequestTypes, jsonObject: JSONObject)

    fun onError(requestTypes: RequestTypes, volleyError: VolleyError)

}
